styles/planet-minecraft/catppuccin.user.css

```diff
@@ -282,9 +282,9 @@
       }
     }
 
-    .content-actions li:not(:only-child, :first-child) {
-      border-top: none;
-    }
```

Comment:
> We shouldn't be disabling borders or changing the layout of the UI in general, only recoloring them. I've commented this out as a way of getting the CI to ping yall, and you can let me know how you want to fix this.

Reply:
Alright, I've gone ahead and checked the classes this effects and it appears something has changed in their upstream causing the specific content actions class to be not styled. Leave commented, I'll work on a fix.

Reply:
Sounds good, can you approve the PR for now? I'll merge it in and you can come back to it whenever.

Reply:
Alright, done. Sorry for late reply.